content-service.go

```diff
@@ -82,7 +82,7 @@ var (
 func startService() ExitCode {
 	serverCfg := conf.GetServerConfiguration()
 	groupsCfg := conf.GetGroupsConfiguration()
-
+	log.Error().Msg("Test Error for glitchtip")
```

Comment:
Please make sure to remove this or there will be a new error every time the service starts :)

Reply:
I mean after it's checked